.gitignore

```diff
@@ -41,3 +41,6 @@ files.json
 project-lock.json
 test/files/environment/projects
 .secrets
+
+user
+auth
```

Comment:
Not a big deal, but make sure you have newlines at the end of your files. A lot of IDE/editors will do it for you.
<img width="644" alt="image" src="https://user-images.githubusercontent.com/9354822/182677457-865d429f-75cd-40d8-a22f-868774b0fcb6.png">

You can also use `prettier` to auto-format your code on save (including final newlines).
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode